### ISSUE TITLE
Summary of license permissions and limitations

### ISSUE THREAD

**Issue by e4drcf** [lbryio/lbry-desktop#6362]
![image](https://user-images.githubusercontent.com/29708735/124243188-3aff9280-db26-11eb-9136-9fef629af510.png)

Github has this feature, maybe that would make sense to publish somewhere on Odysee (video details on watch screen + publishing screen)? 
Users would know which license they want to use while publishing + viewers would know what they can do with content and what they can't

**Comment by JYamihud** [comment]
Exactly. Or at least give us a link to the license text. So people will see it. I have to put an extra CC-BY-SA portion in the publications I make, since selecting CC-BY-SA in the upload menu, doen't show up anywhere in the publication it self.
